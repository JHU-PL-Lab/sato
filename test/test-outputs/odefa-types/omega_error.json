[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "b" ],
          "err_match_val": "false",
          "err_match_expected": "int",
          "err_match_actual": "bool"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "c = a + b",
    "err_steps": 68
  }
]