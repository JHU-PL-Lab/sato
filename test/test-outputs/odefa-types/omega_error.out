No errors found. Further control flows may exist.
